### ISSUE TITLE
Index out-of-range error

### ISSUE THREAD

**Issue by standage** [HadrienG/InSilicoSeq#96]
Thanks for publishing this tool!

I'm testing InSilicoSeq for simulating targeted sequencing of short amplicons. I often run into `string index out of range` error messages when I try to simulate reads from these amplicons. As an example, the command below will elicit the error on [this file](https://github.com/HadrienG/InSilicoSeq/files/3055748/seq4.txt).

```
iss generate --n_reads 500000 --cpus 6 --draft seq4.fasta \
    --model MiSeq --output TESTING --seed 1146345545
```

**Comment by HadrienG** [comment]
solved in `1.3.5`!

**Comment by standage** [comment]
I just encountered this issue again, but with the reverse orientation block. It looks like [the solution](https://github.com/HadrienG/InSilicoSeq/blob/master/iss/error_models/__init__.py#L135-L139) implemented in #97 needs to be applied to the `elif orientation == 'reverse':` block as well. Doing so fixed the issue I had just now.
